app/stories/shared.stories.jsx

```diff
@@ -262,3 +269,74 @@ story('Date Picker')
             preserveLabelSpace
         />
     ))
+
+class SortableListContainer extends React.Component {
+    constructor(props) {
+        super(props)
+
+        this.state = {
+            items: Array(5).fill(true).map((v, i) => `Item #${i}${i === 0 ? ' (Drag me!)' : ''}`),
+        }
+    }
```

Comment:
Remember you can do away with `constructor` entirely:
```js
class SortableListContainer extends React.Component {
    state = {
        items: Array(5).fill(true).map((v, i) => `Item #${i}${i === 0 ? ' (Drag me!)' : ''}`),
    }
    …
}
```

This is the same syntax feature (class field declarations) you're already using for `onSortEnd = ({ newIndex, oldIndex }) => {`